homeassistant/components/onewire/manifest.json

```
@@ -1,7 +1,7 @@
 {
   "domain": "onewire",
   "name": "1-Wire",
-  "codeowners": ["@garbled1", "@epenet"],
+  "codeowners": ["@garbled1", "@epenet", "@szczukot"],
```

You do not need to mark yourself as a codeowner just to make a contribution.
If you plan to make more contributions in the future then I suggest that you make a dedicated PR later.
```suggestion
  "codeowners": ["@garbled1", "@epenet"],
```